Cargo.toml

```diff
@@ -22,6 +22,7 @@ debug = ["clap/debug"]
 no_cargo = ["clap/no_cargo"]
 doc = ["clap/doc"]
 paw = ["structopt-derive/paw"]
+external_doc = ["structopt-derive/external_doc"]
```

Comment:
unstable features might be, well, unstable :) I would like the fact to be reflected in the name of the feature, like `unstable_external_doc` or something so users are aware this is subject to change/break.

Reply:
I agree, and made the change to reflect that